src/UseCases/ApplyForMembership/MembershipApplicationValidator.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\UseCases\ApplyForMembership;
+
+use InvalidArgumentException;
+use WMDE\Fundraising\Frontend\Domain\Model\Euro;
+use WMDE\Fundraising\Frontend\UseCases\ApplyForMembership\ApplicationValidationResult as Result;
+
+/**
+ * @license GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class MembershipApplicationValidator {
+
+	const MIN_YEARLY_PAYMENT_IN_EURO = 25;
```

Comment:
The current minimum yearly fee is €24 for a private person and €100 for institutions/companies.